### ISSUE TITLE
Add namespace to RBAC clusterrole (v0.8+)

### ISSUE THREAD

**Issue by faheem-cliqz** [hjacobs/kube-downscaler#33]
0.8 release added the namespace annotation support for uptime/downtime but didn't add namespace to clusterrole :)

**Comment by hjacobs** [comment]
Good point, thanks!

**Comment by hjacobs** [comment]
Trying out with Minikube and getting the expected error:
```
kube-downscaler-754b9659b6-vmt26 downscaler 2019-02-18 20:26:09,918 INFO: Downscaler v0.8 started with config Namespace(debug=False, default_downtime='never', default_uptime='Mon-Fri 07:30-20:30 CET', dry_run=True, exclude_deployments='kube-downscaler,downscaler', exclude_namespaces='kube-system', exclude_statefulsets='', grace_period=900, interval=60, kind=['deployment'], namespace=None, once=False)
kube-downscaler-754b9659b6-vmt26 downscaler 2019-02-18 20:26:09,918 INFO: **DRY-RUN**: no downscaling will be performed!
kube-downscaler-754b9659b6-vmt26 downscaler 2019-02-18 20:26:09,975 ERROR: Failed to autoscale : namespaces "default" is forbidden: User "system:serviceaccount:default:kube-downscaler" cannot get resource "namespaces" in API group "" in the namespace "default"
kube-downscaler-754b9659b6-vmt26 downscaler Traceback (most recent call last):
kube-downscaler-754b9659b6-vmt26 downscaler   File "/usr/local/lib/python3.7/site-packages/pykube/http.py", line 99, in raise_for_status
kube-downscaler-754b9659b6-vmt26 downscaler     resp.raise_for_status()
kube-downscaler-754b9659b6-vmt26 downscaler   File "/usr/local/lib/python3.7/site-packages/requests/models.py", line 940, in raise_for_status
kube-downscaler-754b9659b6-vmt26 downscaler     raise HTTPError(http_error_msg, response=self)
kube-downscaler-754b9659b6-vmt26 downscaler requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://10.96.0.1:443/api/v1/namespaces/default
kube-downscaler-754b9659b6-vmt26 downscaler 
kube-downscaler-754b9659b6-vmt26 downscaler During handling of the above exception, another exception occurred:
kube-downscaler-754b9659b6-vmt26 downscaler 
kube-downscaler-754b9659b6-vmt26 downscaler Traceback (most recent call last):
kube-downscaler-754b9659b6-vmt26 downscaler   File "/kube_downscaler/main.py", line 40, in run_loop
kube-downscaler-754b9659b6-vmt26 downscaler     dry_run=dry_run, grace_period=grace_period)
kube-downscaler-754b9659b6-vmt26 downscaler   File "/kube_downscaler/scaler.py", line 123, in scale
kube-downscaler-754b9659b6-vmt26 downscaler     default_uptime, default_downtime, forced_uptime, dry_run, now, grace_period)
kube-downscaler-754b9659b6-vmt26 downscaler   File "/kube_downscaler/scaler.py", line 98, in autoscale_resources
kube-downscaler-754b9659b6-vmt26 downscaler     namespace_obj = pykube.Namespace.objects(api).get_by_name(resource.namespace)
kube-downscaler-754b9659b6-vmt26 downscaler   File "/usr/local/lib/python3.7/site-packages/pykube/query.py", line 71, in get_by_name
kube-downscaler-754b9659b6-vmt26 downscaler     self.api.raise_for_status(r)
kube-downscaler-754b9659b6-vmt26 downscaler   File "/usr/local/lib/python3.7/site-packages/pykube/http.py", line 106, in raise_for_status
kube-downscaler-754b9659b6-vmt26 downscaler     raise HTTPError(resp.status_code, payload["message"])
kube-downscaler-754b9659b6-vmt26 downscaler pykube.exceptions.HTTPError: namespaces "default" is forbidden: User "system:serviceaccount:default:kube-downscaler" cannot get resource "namespaces" in API group "" in the namespace "default"

**Comment by hjacobs** [comment]
Fixed in 3e2c482:
```
kube-downscaler-754b9659b6-4n64t downscaler 2019-02-18 20:29:55,936 INFO: Downscaler v0.8 started with config Namespace(debug=False, default_downtime='never', default_uptime='Mon-Fri 07:30-20:30 CET', dry_run=True, exclude_deployments='kube-downscaler,downscaler', exclude_namespaces='kube-system', exclude_statefulsets='', grace_period=900, interval=60, kind=['deployment'], namespace=None, once=False)
kube-downscaler-754b9659b6-4n64t downscaler 2019-02-18 20:29:55,936 INFO: **DRY-RUN**: no downscaling will be performed!
kube-downscaler-754b9659b6-4n64t downscaler 2019-02-18 20:29:55,991 INFO: Scaling down Deployment default/kube-janitor from 1 to 0 replicas (uptime: Mon-Fri 07:30-20:30 CET, downtime: never)
kube-downscaler-754b9659b6-4n64t downscaler 2019-02-18 20:29:55,991 INFO: **DRY-RUN**: would update Deployment default/kube-janitor
kube-downscaler-754b9659b6-4n64t downscaler 2019-02-18 20:29:55,998 INFO: Scaling down Deployment default/nginx from 1 to 0 replicas (uptime: Mon-Fri 07:30-20:30 CET, downtime: never)
kube-downscaler-754b9659b6-4n64t downscaler 2019-02-18 20:29:55,999 INFO: **DRY-RUN**: would update Deployment default/nginx
